etcdserver/server.go

```diff
@@ -145,6 +147,9 @@ type Server interface {
 	// the leader is etcd 2.0. etcd 2.0 leader will not update clusterVersion since
 	// this feature is introduced post 2.0.
 	ClusterVersion() *semver.Version
+
+	// Term returns latest term of raft node.
+	Term() uint64
```

Comment:
This was implemented for `EtcdServer` but not exported. So I added it to interface.